config/drupal.settings.php

```diff
@@ -22,7 +22,7 @@
 // Redis Cache.
 // Due to issues with enabling Redis during install/config import. We cannot enable the cache backend by default.
 // Once you have a site/db installed. Enable the Redis module and change the $redis_enabled to true.
-$redis_enabled = TRUE;
+$redis_enabled = FALSE;
```

Comment:
I always seem to be on the wrong side of this setting. Not quite sure how it'd improve it but it's normally just a headache with new builds rather than established ones.

Reply:
Yes, we found that with this set to TRUE there was always a failure when you first installed. Just at the point that you started the site up for the first time this setting would lead to an error in the log involving Redis that was a bit hard to spot. Setting it to FALSE seems to allow the initial installation to complete successfully and the user can then follow your instructions for enabling Redis after that point.